specification/containerservice/resource-manager/Microsoft.ContainerService/aks/preview/2024-03-02-preview/managedClusters.json

```diff
@@ -9502,6 +9772,164 @@
+        "provisioningState": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The current provisioning state."
+        }
+      },
+      "required": [
+        "name",
+        "primaryAgentPoolName"
+      ]
+    },
+    "LabelSelector": {
+      "type": "object",
+      "description": "A label selector is a label query over a set of resources. The result of matchLabels and matchExpressions are ANDed. An empty label selector matches all objects. A null label selector matches no objects.",
+      "properties": {
+        "matchLabels": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "matchLabels is an array of {key=value} pairs. A single {key=value} in the matchLabels map is equivalent to an element of matchExpressions, whose key field is `key`, the operator is `In`, and the values array contains only `value`. The requirements are ANDed."
+        },
+        "matchExpressions": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/LabelSelectorRequirement"
+          },
+          "description": "matchExpressions is a list of label selector requirements. The requirements are ANDed."
+        }
+      }
+    },
+    "LabelSelectorRequirement": {
+      "type": "object",
+      "description": "A label selector requirement is a selector that contains values, a key, and an operator that relates the key and values.",
+      "properties": {
+        "key": {
+          "type": "string",
+          "description": "key is the label key that the selector applies to."
+        },
+        "operator": {
+          "type": "string",
+          "enum": [
+            "In",
+            "NotIn",
+            "Exists",
+            "DoesNotExist"
+          ],
+          "x-ms-enum": {
+            "name": "operator",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "In",
+                "description": "The value of the key should be in the given list."
+              },
+              {
+                "value": "NotIn",
+                "description": "The value of the key should not be in the given list."
+              },
+              {
+                "value": "Exists",
+                "description": "The value of the key should exist."
+              },
+              {
+                "value": "DoesNotExist",
+                "description": "The value of the key should not exist."
+              }
+            ]
+          },
+          "description": "operator represents a key's relationship to a set of values. Valid operators are In and NotIn"
+        },
+        "values": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "values is an array of string values, the values array must be non-empty."
+        }
+      }
+    },
+    "RebalanceLoadBalancersParameter": {
```

Comment:
Please move this from `definitions` to `parameters`

Reply:
It seems we cannot put body param in parameters.